examples/signature/split.teal

```diff
@@ -13,24 +13,7 @@ global ZeroAddress
 global GroupSize
 int 2
 ==
-bnz l2
-txn CloseRemainderTo
-addr 5MK5NGBRT5RL6IGUSYDIX5P7TNNZKRVXKT6FGVI6UVK6IZAWTYQGE4RZIQ
-==
-txn Receiver
-global ZeroAddress
-==
-&&
-txn Amount
-int 0
-==
-&&
-txn FirstValid
-int 3000
->
-&&
-b l3
-l2:
+bz l2
```

Comment:
Are these auto-generated from the pyteal file?

Reply:
Yep, it seems Shiqi updated `split.teal` and `dutch_auction.teal` with new generated version.